elisa/src/client/RunDetails/components/PlotButtonBar.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { PureComponent } from 'react';
+import $ from 'jquery';
+import { getServerContext } from "@labkey/api";
+
+import { PlotOptions } from "../models";
+import { getPlotTitle, getResultsViewURL } from "../utils";
+import { exportSVGToFile } from "../actions";
+
+interface Props {
+    protocolId: number,
+    runId: number,
+    runPropertiesRow: {[key: string]: any},
+    plotOptions: PlotOptions,
+    plotId: string
+}
+
+export class PlotButtonBar extends PureComponent<Props> {
+
+    getSVGElement() {
+        return $('#' + this.props.plotId + '>svg');
```

Comment:
Like mentioned above, wonder if this could be done with React.createRef instead of using jQuery

Reply:
Good call, done. We should likely switch the biologics usage like this at some point as well.